test_flake8 doesn't work with flake8 3.8

flake8 3.8 changes the Application API such that test_flake8 no longer works:

```
=================================================================== FAILURES ===================================================================
_________________________________________________________________ test_flake8 __________________________________________________________________

    def test_flake8():
        configure_logging(1)
        argv = [
            '--extend-ignore=' + ','.join([
                'A003', 'D100', 'D101', 'D102', 'D103', 'D104', 'D105', 'D107']),
            '--exclude', 'vcstool/compat/shutil.py',
            '--import-order-style=google']
>       style_guide = get_style_guide(argv)

test/test_flake8.py:19: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

argv = ['--extend-ignore=A003,D100,D101,D102,D103,D104,D105,D107', '--exclude', 'vcstool/compat/shutil.py', '--import-order-style=google']

    def get_style_guide(argv=None):
        # this is a fork of flake8.api.legacy.get_style_guide
        # to allow passing command line argument
        application = Application()
>       application.parse_preliminary_options_and_args([])
E       AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'

test/test_flake8.py:40: AttributeError
=========================================================== short test summary info ============================================================
FAILED test/test_flake8.py::test_flake8 - AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
======================================================== 1 failed, 16 passed in 14.55s =========================================================
```

Consider pinning to 3.7.9 or updating the test.

--- a/test/test_flake8.py
+++ b/test/test_flake8.py
@@ -37,11 +37,25 @@ def get_style_guide(argv=None):
     # this is a fork of flake8.api.legacy.get_style_guide
     # to allow passing command line argument
     application = Application()
-    application.parse_preliminary_options_and_args([])
-    application.make_config_finder()
-    application.find_plugins()
-    application.register_plugin_options()
-    application.parse_configuration_and_cli(argv)
+    if hasattr(application, 'parse_preliminary_options'):
+        prelim_opts, remaining_args = application.parse_preliminary_options(
+            argv)
+        from flake8 import configure_logging
+        configure_logging(prelim_opts.verbose, prelim_opts.output_file)
+        from flake8.options import config
+        config_finder = config.ConfigFileFinder(
+            application.program, prelim_opts.append_config,
+            config_file=prelim_opts.config,
+            ignore_config_files=prelim_opts.isolated)
+        application.find_plugins(config_finder)
+        application.register_plugin_options()
+        application.parse_configuration_and_cli(config_finder, remaining_args)
+    else:
+        application.parse_preliminary_options_and_args([])
+        application.make_config_finder()
+        application.find_plugins()
+        application.register_plugin_options()
+        application.parse_configuration_and_cli(argv)
     application.make_formatter()
     application.make_guide()
     application.make_file_checker_manager()

--- a/vcstool/clients/git.py
+++ b/vcstool/clients/git.py
@@ -28,7 +28,7 @@ class GitClient(VcsClientBase):
         if not command.all and not result['returncode']:
             # only show current branch
             lines = result['output'].splitlines()
-            lines = [l[2:] for l in lines if l.startswith('* ')]
+            lines = [line[2:] for line in lines if line.startswith('* ')]
             result['output'] = '\n'.join(lines)
 
         return result

